README.md

```diff
@@ -129,6 +129,11 @@ Click the button below to start deployment.
    **Note:** This value defaults to `insight-operational-logs`. This Event Hub is created automatically by Azure when a subscription [Log Profile is integrated with Event Hub through the Azure Monitor service](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/stream-monitoring-data-event-hubs#azure-subscription-monitoring-data).
    Follow this guide to [Stream the Azure Activity Log to Event Hubs](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/activity-logs-stream-event-hubs).
 
+   - **Event Hub Retention** - Type the number of data retention days between 1 and 7. The value applies only if new event hub is created.
+   - **Event Hub Partition Count** - Type the number of partitions between 2 and 32. The value applies only if new event hub is created.
+
+   **Note:** Partitions let you scale for your downstream processing. The partition count is not changeable, please consider long-term scale when setting partition count. General recommendation is to balance 1:1 throughput units and partitions to achieve optimal scale. There is no charge for the number of partitions you have within an Event Hub. Please make sure to follow the [guideline](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-features#partitions-1) and [FAQ](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-faq#how-many-partitions-do-i-need) before setting the partition count.
```

Comment:
Perhaps add 'once the Event Hub is created' after 'is not changeable'?

Reply:
Also: "This is why we strongly recommend that the number of partitions in an event hub be greater than or equal to the number of throughput units." (from the same page)
So, do we really require 1:1 (since I'm guessing this also depends on collector performance).

Reply:
So, perhaps we should just default this to 32 and leave it at that (that is, not add a parameter even)?

Reply:
I think this is the point of this change is to give customers better control over event hub scaling and at the same time use recommended values by default.